test/src/index/transport/testMobileWebSupport.js

```diff
@@ -0,0 +1,241 @@
+const assert = require('assert');
+const common = require('../../../common.js');
+const path = require('path');
+const CommandGlobals = require('../../../lib/globals/commands.js');
+const MockServer = require('../../../lib/mockserver.js');
+const {settings} = common;
+const {runTests} = common.require('index.js');
+const Transport = require('../../../../lib/transport/selenium-webdriver/index.js');
```

Comment:
use `common.require()` here

Reply:
Fixed in [5b5bf9f](https://github.com/nightwatchjs/nightwatch/pull/3572/commits/5b5bf9f1ff7dd92182d5c4329839374912bd3630)